ga/23.0.0.6/full/Dockerfile.ubi.ibmjava8

```diff
@@ -0,0 +1,53 @@
+# (C) Copyright IBM Corporation 2023.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG PARENT_IMAGE=icr.io/appcafe/websphere-liberty:kernel-java8-ibmjava-ubi
```

Comment:
Although it's fine to use the Liberty version-less tag for now, it should be set explicitly on _full_ images in order to ensure the right kernel image is used as the base. Otherwise, when the _latest_ moves to 23.0.0.7, this will use the wrong base.
For example, `icr.io/appcafe/websphere-liberty:23.0.0.6-kernel-java8-ibmjava-ubi`

Reply:
Fixed